### PR TITLE
minor modification in test

### PR DESCRIPTION
**Pull request by nurali-techie**
Ref: https://github.com/fabric8-services/fabric8-notification/pull/95

### CODE REVIEW

File: collector/cve_test.go [hunk 1]
```diff
@@ -40,7 +43,8 @@ func TestCVEResolver(t *testing.T) {
 
 	cveResolver := collector.NewCVEResolver(authClient, witClient)
 	codebaseURL := "git@github.com:testrepo/testproject1.git"
-	recvs, _, err := cveResolver(context.Background(), codebaseURL)
+	ctx, _ := client.ContextWithRequestID(context.Background())
+	recvs, _, err := cveResolver(ctx, codebaseURL)
```
Comment (dipak-pawar) [inline collector/cve_test.go]: I was wondering about the test which is verifying 500 from wit for missing requestID. Do we have it already?

Comment (nurali-techie) [reply collector/cve_test.go]: not getting you.

Comment (dipak-pawar) [reply collector/cve_test.go]: Do we have some tests where WIT it returning 500 for missing header?

Comment (nurali-techie) [reply collector/cve_test.go]: now, I changed to 400.  Is this what your concern ??

Comment (dipak-pawar) [reply collector/cve_test.go]: it's good now @nurali-techie

**Pull request merged**